django/db/models/expressions.py

```diff
@@ -1332,7 +1332,9 @@ def get_db_converters(self, connection):
 class ColPairs(Expression):
     def __init__(self, alias, targets, sources, output_field):
         super().__init__(output_field=output_field)
-        self.alias, self.targets, self.sources = alias, targets, sources
+        self.alias = alias
+        self.targets = targets
+        self.sources = sources
```

Comment:
You used this pattern in other places, e.g.
```python
col, val = next(cols_iter), next(vals_iter)
lookup, connector = next(lookups), next(connectors)
```
I don't think it's worth changing, but it's something you should avoid in the future.

Reply:
Okay well I just liked the way it looked but if it's an issue then let's fix it.